lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -1488,7 +1488,7 @@ define(['backbone', 'jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers
 
                         topicsJson = {
                             course_wide_discussions: {
-                                children: ['Topic_C_1'],
+                                children: [['Topic_C_1', 'entry']],
```

Comment:
We don't have any one defined place for them as far as I know. If you're sharing them between two files a `constants.js` at the root of whatever directory tree both files share sounds good to me.